### PR TITLE
Fix Windows support

### LINKED ISSUE #3: Incorrect autofixing behaviour
Thank you for implementing this awesome library!

It produces incorrect result in this case, however:

Original value:
```ts
export { something } from './src/client'
```

After autofixing:
```ts
export { something } from './src\client.js'
```

Slash shouldn't be changed when adding extension.

Note: I'm using Windows, but I'm not sure if this is affecting this behaviour

### PR DESCRIPTION
**Pull request by kibertoad**
fixes #3

### CODE REVIEW

**Comment by satazor** [comment]
Looks good. Let’s see if tests pass

**Comment by kibertoad** [comment]
@satazor master is currently broken :-/

**Comment by satazor** [comment]
Fixed it. Made the commit from my phone :D

**Comment by satazor** [comment]
Can you rebase to see if checks run?

**Review by satazor — APPROVED** [review]

**Comment by satazor** [comment]
I think we need to use matrix https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs for windows

**Comment by kibertoad** [comment]
@satazor Can you approve the workflow run again? Seems like explicit matrix was needed

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,18 +1,25 @@
 name: Node CI
 
-on: [push]
+on:
+  pull_request:
```
Comment (satazor) [inline .github/workflows/ci.yml:4]: CI seems fixed. Should we revert this?

**Comment by satazor** [comment]
Tests seem to be failing on windows 😅

**Comment by kibertoad** [comment]
@satazor Linting, to be more precise: https://github.com/satazor/eslint-plugin-import-esm/actions/runs/7775528461/job/21201639760?pr=4

And it is passing locally. I don't understand what it is complaining about and why. Any ideas?

**Comment by satazor** [comment]
I will check in a few minutes when I arrive home

**Comment by satazor** [comment]
Could you try bumping `eslint-plugin-import-esm`? If it still fails, I guess we can skip the linting on windows in the workflow file.

**Comment by kibertoad** [comment]
@satazor Do you mean bumping "version" in package.json?

**Comment by satazor** [comment]
Yes to `1.6.2`

**Comment by kibertoad** [comment]
@satazor it's 1.2.1 now, why 1.6.2?

**Comment by satazor** [comment]
There are entries on the changelog of the tool that might be related to the issue.

**Comment by kibertoad** [comment]
@satazor It is passing now, except for Node 14, which returns garbage.

**Comment by satazor** [comment]
Yea let's remove support for older node js versions. Let's just support node lts: >= 18

**Comment by satazor** [comment]
Can you change package.json engines field to be only node >= 18?

**Review by satazor — APPROVED** [review]

**Comment by kibertoad** [comment]
@satazor What's missing until this can be merged and released?

**Comment by satazor** [comment]
Will release shortly

**Pull request merged**